man/systemd.network.xml

```diff
@@ -1254,6 +1254,15 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             unset.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Type=</varname></term>
+          <listitem>
+            <para>Specifies Routing Policy Database (RPDB) rule type.The syntax and semantics are the same as for <varname>Type=</varname> in the
+             [Route] section described above. Defaults to unset.
```

Comment:
Spurious indentation?

Default seems to be unicast. See `sd_rtnl_message_new_routing_policy_rule()`.